packages/pn-personafisica-webapp/src/component/Contacts/SpecialContactElem.tsx

```diff
@@ -146,17 +152,22 @@ const SpecialContactElem = memo(({ address, senders, recipientId }: Props) => {
               },
             ]}
             saveDisabled={!!formik.errors[f.id]}
-            removeModalTitle={t(`${f.labelRoot}.remove-${addressTypeToLabel[f.addressId]}-title`, { ns: 'recapiti' })}
+            removeModalTitle={t(`${f.labelRoot}.remove-${addressTypeToLabel[f.addressId]}-title`, {
+              ns: 'recapiti',
+            })}
             removeModalBody={t(`${f.labelRoot}.remove-${addressTypeToLabel[f.addressId]}-message`, {
               value: formik.values[f.id],
               ns: 'recapiti',
             })}
             value={formik.values[f.id]}
             onConfirmClick={updateContact}
```

Comment:
This will become
`onConfirmClick={(status) => updateContact(status, f.id)}`

Reply:
do it